Dockerfile

```diff
@@ -1,45 +1,58 @@
+# using tag debian:buster-slim to automatically receive updates on buster
+# assuming debian:buster-slim is stable enough, so on breaking changes
 FROM debian:buster-slim as builder
 
 ENV DEBIAN_FRONTEND noninteractive
-ENV TZ Europe/Amsterdam 
-ENV LIGHTTPD_VERSION=1.4.63
+ENV TZ Europe/Amsterdam
+ENV LIGHTTPD_VERSION=1.4.65
 
 # https://redmine.lighttpd.net/projects/lighttpd/wiki/DevelGit
 RUN apt-get -y update && \
     apt-get install -y --no-install-recommends \
-    ca-certificates \
-    make \
-    autoconf \
-    automake \
-    pkg-config \
-    libtool \
-    m4 \
-    git \
-    libssl-dev \
-    libpcre3-dev \
-    zlib1g-dev \
-    libbz2-dev \
-    liblua5.2-dev \
-  && rm -rf /var/lib/apt/lists/*
+      autoconf \
+      automake \
+      ca-certificates \
+      git \
+      libbz2-dev \
+      liblua5.2-dev \
+      libpcre2-dev \
+      libssl-dev \
+      libtool \
+      m4 \
+      make \
+      pkg-config \
+      zlib1g-dev &&\

```

Comment:
\<zeik-modus\>ik zou de `&&` op de nieuwe regel zetten. zodat als er zo'n regel uit gaat of bijkomt niet de vorige regel ook in de diff zit\</zeik-modus\>

Reply:
terecht